src/test/java/create_task_use_case/CreateTaskDataAccessTest.java

```diff
@@ -0,0 +1,31 @@
+package create_task_use_case;
+
+import data_access.DataAccessDay;
+import data_access.DataAccessTask;
+import org.junit.jupiter.api.Test;
+
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CreateTaskDataAccessTest {
+    @Test
+    public void testExistsByTitleWithoutDuplicateTask(){
+        CreateTaskDataAccess createTaskDataAccess = new CreateTaskDataAccess();
+        boolean actual = createTaskDataAccess.existsByTitle("Check for task", 0);
+        assertFalse(actual);
+    }
+    @Test
+    public void testExistsByTitleWithDuplicateTask(){
```

Comment:
Once again, please explicitly check if the 'days' instance attribute of CreateTaskDataAccess has been mutated.

Reply:
why in this file? The setup in this test case is directly mutating the 'days' class attribute already.

Reply:
The point of this test is to check if the title exists, works properly. existsByTitle does not have actual functionality for saving/mutation as the save method for dataAccess

Reply:
To clarify my comment here, I was responding directly to Raghav. I don't think anything needs to be changed here.

Reply:
That makes sense Daniel. Thanks.